src/main/resources/plugin.yml

```
@@ -70,7 +75,18 @@ permissions:
       lumaguilds.guild.war: true
       lumaguilds.guild.info: true
       lumaguilds.guild.history: true
+      lumaguilds.admin.emoji.custom:
+    description: Set custom emoji for a Level 100 guild
```

YAMLlint (1.38.0)

[error] duplication of key "description" in mapping

(key-duplicates)